src/Gang/Gang.ts

```diff
@@ -104,50 +103,56 @@ export class Gang {
     }
   }
 
-  processGains(numCycles = 1): void {
-    // Get gains per cycle
-    let moneyGains = 0;
-    let respectGains = 0;
-    let wantedLevelGains = 0;
+  /** Process respect/wanted/money gains
+   * @param numCycles The number of cycles to process. */
+  processGains(numCycles: number): void {
+    let moneyGainPerCycle = 0;
+    let wantedLevelGainPerCycle = 0;
+    let respectGainsTotal = 0;
+    /** Number of members performing actions that lower wanted level */
     let justice = 0;
-    for (let i = 0; i < this.members.length; ++i) {
-      respectGains += this.members[i].earnRespect(numCycles, this);
-      moneyGains += this.members[i].calculateMoneyGain(this);
-      const wantedLevelGain = this.members[i].calculateWantedLevelGain(this);
-      wantedLevelGains += wantedLevelGain;
-      if (this.members[i].getTask().baseWanted < 0) justice++; // this member is lowering wanted.
+
+    for (const member of this.members) {
+      respectGainsTotal += member.earnRespect(numCycles, this);
+      moneyGainPerCycle += member.calculateMoneyGain(this);
+      wantedLevelGainPerCycle += member.calculateWantedLevelGain(this);
+      if (member.getTask().baseWanted < 0) justice++;
     }
-    this.respectGainRate = respectGains;
-    this.wantedGainRate = wantedLevelGains;
-    this.moneyGainRate = moneyGains;
-    const gain = respectGains;
-    this.respect += gain;
+
+    this.respectGainRate = respectGainsTotal / numCycles;
+    this.wantedGainRate = wantedLevelGainPerCycle;
```

Comment:
Why aren't these adjusted?

Reply:
they are already per cycle and multiplied by `numCycles` later :)